sqlalchemy_api_handler/mixins/has_activities_mixin.py

```diff
@@ -36,15 +36,15 @@ def _get_activity_join_by_entity_id_filter(self):
         return ((Activity.old_data[id_key].astext.cast(BigInteger) == id_value) | \
                 (Activity.changed_data[id_key].astext.cast(BigInteger) == id_value))
 
-    def _get_activity_join_filter(self):
+    def get_activity_join_filter(self):
```

Comment:
J'aurais plutôt vu un nommage comme :
```suggestion
    def join_self_activities(self):
```
ou quelque chose dans le genre, t'en penses quoi ?

Reply:
:+1: